capstone/src/main/webapp/script.js

```diff
@@ -47,5 +48,97 @@ function  formatCardInHTML(studySet) {
           <span> By  ${studySet.user_author} from ${studySet.university}</span>
          </div>
       </div>
-    </a>`;
+   </a>`;
+}
+
+//View Study Set
+async function loadStudySetToPage() {
+  var urlParams = new URLSearchParams(window.location.search);
+  const studySetId = urlParams.get("id");
+  const response = await fetch("/study_set/" + studySetId);
+  const studySet = await response.json();
+
+  document.getElementById(
+    "studySetDirectory-container"
+  ).innerHTML = configureStudySetCardDirectoryHTML(studySet.cards);
+
+  document.getElementById(
+    "studySetDetails-container"
+  ).innerHTML = configureStudySetDetailsHTML(studySet);
+
+  document.getElementById(
+    "allCards-container"
+  ).innerHTML = configureAllCardsHTML(studySet.cards);
+}
+
+function configureStudySetCardDirectoryHTML(cards) {
+  var configuredCardList = "";
+  for (var card of cards) {
+    configuredCardList += `
+    <li class="truncate">
+      <a onclick="displayCard('${ card.front}', '${card.back}')" class="btn-flat">
```

Comment:
This will probably break if front or back contains single quotes. Okay for this PR but should be addressed in https://trello.com/c/f4iGDzLW/36-app-should-protect-users-from-xss-attacks

Reply:
to be addressed in different PR